lib/msal-node/test/TestConstants.ts

```diff
@@ -0,0 +1 @@
+export const TESTS = 100;
```

Comment:
I feel like this isn't specific enough.

Reply:
Can change the name, it is mostly stress testing codes we generate.

Reply:
Removed